web-app/HeimScripts/heatmap/summary.R

```diff
@@ -147,6 +198,9 @@ produce_boxplot <- function(measurement_tables, phase)
     measurement_vectors[[i]] <- unlist(measurement_vectors[[i]]) 
```

Comment:
nodes <- gsub("_.*","",names(measurement_tables))
      subsets <- gsub(".*_","", names(measurement_tables))
Are nodes already included in the input?